addon/components/ember-thead/component.js

```diff
@@ -333,11 +347,11 @@ export default Component.extend({
     this.onUpdateSorts?.(newSorts);
   },
 
-  fillupHandler() {
+  fillupHandler(isInitialRun = false) {
```

Comment:
It seems like we always call `fillupHandler` w/ `true` argument. Is that right?

Reply:
The table resize sensor calls it with a `false` argument, but I'll double check if it's actually necessary to make the distinction.

Reply:
Eliminated argument in favor of a distinct `performInitialLayout()` method called on init.